napari_assistant/_gui/_category_widget.py

```diff
@@ -33,7 +33,7 @@
 OP_NAME_PARAM = "op_name"
 OP_ID = "op_id"
 DEFAULT_BUTTON_SIZE = 40
-
+LAYER_NAME_PREFIX = "Result of "
```

Comment:
Great, thanks for this! Very good style

Reply:
No problem! I'm trying to do this as much as possible since I've shot myself in the foot (so to speakt) too many times now :)